media/markdown_set_module_dependency.md

```diff
@@ -0,0 +1,23 @@
+### Set module dependency locations
+
+
+Indicate the Inmanta extension where to look for dependency modules when working on an individual module.
+
+When working on a project, this option is ignored and the Inmanta extension will look for modules in the repositories and pip indexes defined in the repo section of the [project.yaml](https://docs.inmanta.com/community/latest/reference/projectyml.html#project-yml) file.
+
+These module dependencies follow this scheme:
+
+```json
+    {"type": "git", "url": "https://github.com/inmanta/"}
+```
+
+The type property can have two values:
+# Type "git"
+
+The url property holds the URI of the parent location for a v1 module. (Organization url for github, Group url for gitlab, parent folder for a local repository...)
+
+
+# Type "package"
+
+
+The url property holds the pip index url in which to look for v2 modules. ⚠️ Be mindful of [dependency confusion attacks](https://www.websecuritylens.org/how-dependency-confusion-attack-works-and-how-to-prevent-it/) when setting this up: make sure you have full control over this index.
```

Comment:
I think we can be a bit less alarming here. It's good to point this out, but let's make sure to make clear it's only really an issue if you use more than one index. If we want to keep it short we could just say there are potential security risks when setting more than one index and point to https://docs.inmanta.com/community/dev/reference/projectyml.html#inmanta.module.ProjectMetadata for more information. I have no strong preference between that and just expanding on this to make it more clear.